ModelBundle/DataFixtures/MongoDB/LoadRouteDocumentData.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\DataFixtures\MongoDB;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\ModelBundle\Document\RouteDocument;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraFunctionalFixturesInterface;
+
+/**
+ * Class LoadRouteDocumentData
+ */
+class LoadRouteDocumentData implements FixtureInterface, OrchestraFunctionalFixturesInterface
+{
+    /**
+     * Load data fixtures with the passed EntityManager
+     *
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager)
+    {
+        $patterns = array(
+            'foo',
+            'foo/{bar}',
+            'foo/bar',
+            'baz/bar',
+            'foo/{bar}/baz'
```

Comment:
Wouldn't it make the database more consistent if the path were matching the node functionnal fixtures ?